internal/catalog/internal/types/failover_policy.go

```diff
@@ -174,13 +174,19 @@ func ValidateFailoverPolicy(res *pbresource.Resource) error {
 func validateFailoverConfig(config *pbcatalog.FailoverConfig, ported bool, wrapErr func(error) error) error {
 	var merr error
 
+	if config.SamenessGroup != "" {
+		// TODO(v2): handle other forms of failover
```

Comment:
Rather than figure out how to make this work at this time, I left the fields alone but I made it impossible to set them for now.